keps/prod-readiness/sig-apps/3998.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3998
+beta:
```

Comment:
I think you need alpha first?

Reply:
Yes, we should start this enhancement as an alpha stage.

Reply:
done.